roachtest: increase the verbosity of runMultiStoreRemove

iskettaneh opened the pull request:
Recent failures in runMultiStoreRemove seems to indicate a potential race between follower replica removal and leader fortification.

References: #153517

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/153680)
<!-- Reviewable:end -->

iskettaneh committed:
roachtest: increase the verbosity of runMultiStoreRemove

Recent failures in runMultiStoreRemove seems to indicate a potential
race between follower replica removal and leader fortification.

References: #153517

Release note: None

iskettaneh commented:
TFTR!

bors r+

iskettaneh commented:
blathers backport 25.3 25.2 25.1 24.3
